Userland/Libraries/LibPDF/Command.h

```diff
@@ -70,6 +106,8 @@ class Command {
 
         if (symbol_string == "'")
             return CommandType::TextNextLineShowString;
+        if (symbol_string == "''")
+            return CommandType::TextNextLineShowStringSetSpacing;
```

Comment:
Is there a way I could put these in the macro enumeration above? It would require something like `V(whatever, whatever, ')`, and of course that hanging apostrophe causes issues. I always thought macros were just copy-paste, but I suppose that's not quite accurate.

Reply:
You could quote them all?

Reply:
Unfortunately that gives me `"\"'\""`. Oh well, not that big of a deal

Reply:
Yeah because you're stringifying it later, you're gonna have to quote all the items in that macro and get rid of the stringification